src/helpers/fetch.js

```diff
@@ -73,6 +73,14 @@ class Fetch {
   _handleResponse(result) {
     return result.json().then(json => {
 
+
+      fs.appendFileSync('/Users/andreyluchianic/Documents/log.txt', JSON.stringify({
```

Comment:
I don't really think I still have this path.

Reply:
fixed